_posts/2018-10-12-using-sass-maps-to-extend-bootstrap.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "Using Sass maps to extend bootstrap"
```

Comment:
How about "Using Sass Maps to Extend Bootstrap"?

Reply:
i'm ok either way; pr'd